lib/LDAPUserManager.php

```diff
@@ -233,14 +233,19 @@ public function createUser($username, $password) {
 		$this->ensureAttribute($newUserEntry, $displayNameAttribute, $username);
 
 		$ret = ldap_add($connection, $newUserDN, $newUserEntry);
-		$message = $ret
-			? 'Create LDAP user \'{username}\' ({dn}'
-			: 'Unable to create LDAP user \'{username}\' ({dn})';
-		$this->logger->error($message, [
+
+		$message = 'Create LDAP user \'{username}\' ({dn}';
```

Comment:
closing bracket missing

Reply:
good catch

Reply:
i let the CI run before pushing the typo fix. We have long queues today…

Reply:
green :)